supabase/migrations/20230404202715_add_project_columns.sql

```diff
@@ -0,0 +1,22 @@
+alter table public.projects 
+alter COLUMN "id" type text,
+add COLUMN "handle" text,
+add COLUMN "projectId" int,
+add COLUMN "pv" char(1) check (pv in('1', '2')),
```

Comment:
but @aeolianeth i think we can use this instead of enum?

Reply:
got it, i think enum prolly the best practice but idm. whatever works!

Reply:
Sorry missed this comment: https://github.com/jbx-protocol/juice-interface/pull/3334#discussion_r1157610912

LGTM!